### PR TITLE
nba boxplot ready for code review

### PR DESCRIPTION
**Pull request by carlinakim**

### CODE REVIEW

File: tests/test_nba_boxplot.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
+from pysketball import nba_boxplot
+import pandas as pd
+import numpy as np
+from pytest import raises
+
```
Comment (AndresPitta) [inline tests/test_nba_boxplot.py:7]:
```suggestion
   import altair as alt
   
```

File: tests/test_nba_boxplot.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    with raises(ValueError, match= "Empty arguments, choose either position or teams argument for plot"):
+        nba_boxplot.nba_boxplot(nba_2018, stats= "GP", position= None, teams= None)
+        
+    with raises(TypeError, match= "stats argument has to be numeric"):
+        nba_boxplot.nba_boxplot(nba_2018, stats= "Team", position= "POS")
+        
+
+    assert isinstance(nba_boxplot.nba_boxplot(nba_2018, stats= "GP", teams= ("ORL", "UTAH", "LAC"))[1], alt.vegalite.v4.api.LayerChart)
+
+    assert isinstance(nba_boxplot.nba_boxplot(nba_2018, stats= "GP", position= 'POS', teams= None)[1], alt.vegalite.v4.api.LayerChart)
```
Comment (AndresPitta) [inline tests/test_nba_boxplot.py]:
```suggestion


    assert isinstance(nba_boxplot.nba_boxplot(nba_2018, stats= "GP", teams= ("ORL", "UTAH", "LAC")), alt.vegalite.v4.api.Chart)

    assert isinstance(nba_boxplot.nba_boxplot(nba_2018, stats= "GP", position= 'POS', teams= None), alt.vegalite.v4.api.Chart)
```

File: tests/test_nba_boxplot.py (hunk 1 above)
Comment (carlinakim) [reply tests/test_nba_boxplot.py:7]: added these thanks!

File: tests/test_nba_boxplot.py (hunk 2 above)
Comment (carlinakim) [reply tests/test_nba_boxplot.py]: fixed!

**Review by vanandsh — CHANGES_REQUESTED** [review]
Everyone looks good! Only one change is needed - Please remove the Untitled.ipynb that got pushed with the other files.

**Review by vanandsh — APPROVED** [review]
All good :) Thanks @carlinakim

**Pull request merged**